tasks/interactive-bots.md

```diff
@@ -168,14 +180,15 @@ messages to users.
                                          'from %s:' % (original_sender,))
   ```
 
+
 * Create a links bot in settings for an existing user. Use its credentials in
 `~/.zuliprc-local`. Subscribe the bot to `devel` and `social` streams.
 
 * Check that the bot is working as expected:
 
   - it reacts to the `@link` and `@resource` at the beginning of a message
 
-  - it currently posts the message to the followup stream, as the followup bot
+  - it currently posts the message to the links stream, as the followup bot
```

Comment:
So we have copied the followup bot right, therefore the bot is currently posting to the followup stream. So I think you shouldn't change this line, as @trueskawka mentioned.